docs/creature_text_locale.md

```diff
@@ -8,37 +8,46 @@ This table is used to provide to localized clients with localized string for cre
 
 **Structure**
 
-| Field                                        | Type         | Attributes | Key | Null | Default | Extra | Comment |
-|----------------------------------------------|--------------|------------|-----|------|---------|-------|---------|
-| [entry](#creature_text_locale-entry)         | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [groupid](#creature_text_locale-groupid)     | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [id](#creature_text_locale-id)               | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [text\_loc1](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc2](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc3](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc4](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc5](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc6](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc7](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc8](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
+| Field           | Type         | Attributes | Key      | Null     | Default | Extra | Comment |
+|-----------------|--------------|------------|----------|----------|---------|-------|---------|
+| [CreatureID][1] | mediumint(8) | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [GroupID][2]    | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [ID][3]         | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [Locale][4]     | varchar(4)   |            |  PRI     |  NO      |         |       |         |
+| [Text][5]       | text         |            |          |  YES     |         |       |         |
+
+[1]: #CreatureID
+[2]: #GroupID
+[3]: #ID
+[4]: #Locale
+[5]: #Text
```

Comment:
Only small letters in the [1]: etc, otherwise it does not work

Reply:
> Only small letters in the [1]: etc, otherwise it does not work

I forgot about this, ok. I'll modify it now.